<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mySetting</class>
 <widget class="QWidget" name="mySetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/set/res/setting1.png);</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>471</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 14pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_1">
    <attribute name="title">
     <string> 主题设置</string>
    </attribute>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>91</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
font: 87 11pt &quot;Arial Black&quot;;</string>
     </property>
     <property name="text">
      <string>默认</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>91</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 11pt &quot;Arial Black&quot;;</string>
     </property>
     <property name="text">
      <string>自定义</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>82</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Arial&quot;;
font: 87 11pt &quot;Arial Black&quot;;</string>
     </property>
     <property name="text">
      <string> 背景图片</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Arial&quot;;
font: 87 11pt &quot;Arial Black&quot;;</string>
     </property>
     <property name="text">
      <string> 宝石图片</string>
     </property>
    </widget>
    <widget class="QPushButton" name="choose1">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>80</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 11pt &quot;Arial Black&quot;;</string>
     </property>
     <property name="text">
      <string> 浏览...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="choose2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>130</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 11pt &quot;Arial Black&quot;;</string>
     </property>
     <property name="text">
      <string> 浏览...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>-30</x>
       <y>-70</y>
       <width>561</width>
       <height>441</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/new/prefix1/setting1.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_4</zorder>
    <zorder>radioButton</zorder>
    <zorder>radioButton_2</zorder>
    <zorder>label</zorder>
    <zorder>label_2</zorder>
    <zorder>choose1</zorder>
    <zorder>lineEdit</zorder>
    <zorder>lineEdit_2</zorder>
    <zorder>choose2</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>音乐设置</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>371</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 11pt &quot;Arial Black&quot;;</string>
     </property>
     <property name="title">
      <string>音效设置</string>
     </property>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>73</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string> 音效</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>30</y>
        <width>91</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string> 背景音乐</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>371</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 11pt &quot;Arial Black&quot;;</string>
     </property>
     <property name="title">
      <string> 背景音乐设置</string>
     </property>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>91</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>默认</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_4">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>30</y>
        <width>91</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string> 自定义</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="choose3">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>70</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string> 浏览...</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>-30</x>
       <y>-70</y>
       <width>761</width>
       <height>521</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/new/prefix1/setting1.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_5</zorder>
    <zorder>groupBox</zorder>
    <zorder>groupBox_2</zorder>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>320</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 14pt &quot;Arial Black&quot;;
font: 87 12pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string> 确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>320</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 12pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string> 取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>530</width>
     <height>369</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/set/res/setting1.png);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_3</zorder>
  <zorder>tabWidget</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>pushButton_3</zorder>
 </widget>
 <resources>
  <include location="set1.qrc"/>
 </resources>
 <connections/>
</ui>
